Firestore/Source/Local/FSTRemoteDocumentChangeBuffer.mm

```diff
@@ -47,42 +51,41 @@ + (instancetype)changeBufferWithCache:(id<FSTRemoteDocumentCache>)cache {
 - (instancetype)initWithCache:(id<FSTRemoteDocumentCache>)cache {
   if (self = [super init]) {
     _remoteDocumentCache = cache;
-    _changes = [NSMutableDictionary dictionary];
+    _changes = absl::make_unique<std::map<DocumentKey, FSTMaybeDocument *>>();
```

Comment:
Optional: I don't think we support any platforms that don't have `std::make_unique` (Visual Studio has it since 2013, if I'm not mistaken).

Reply:
Some platforms have it but don't provide it unless you compile in C++14 mode and we target C++11.

Reply:
Thanks, good to know.